.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,21 +1,19 @@
 ---
 name: Bug report
-about: Create a report to help us improve
-title: ''
-labels: bug
+about: Create a bug report to help us improve
+title: "Short clear titles please"
+labels: ["bug", "triage"]
 assignees: ''
 
 ---
 
 **Describe the bug**
 A clear and concise description of what the bug is.
 
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+**Sample code**
```

Comment:
Should we maybe use the same format as the feature request?

```suggestion
## Sample code
```

e.g. headings, instead of bold text?